src/Console/ConsoleOptionParser.php

```diff
@@ -938,13 +936,10 @@ protected function _parseArg(string $argument, array $args): array
             throw new ConsoleException('Too many arguments.');
         }
 
-        if ($this->_args[$next]->validChoice($argument)) {
-            $args[] = $argument;
-
-            return $args;
-        }
+        $this->_args[$next]->validChoice($argument);
+        $args[] = $argument;
 
-        throw new ConsoleException('Invalid choice for argument: ' . $argument);
```

Comment:
Don't we need this exception to signal that an invalid choice was made for the option?

Reply:
The `ConsoleInputArgument::validChoice()` already throws an exception.